google/cloud/spanner/commit_options.h

```diff
@@ -36,6 +37,18 @@ class CommitOptions {
   /// Default options: no stats.
   CommitOptions() = default;
 
+  /**
+   * Constructs from the the new, recommended way to represent options
+   * of all varieties, `google::cloud::Options`.
+   */
+  explicit CommitOptions(Options const& opts);
+
+  /**
+   * Converts to the new, recommended way to represent options of all
+   * varieties, `google::cloud::Options`.
+   */
+  explicit operator Options() const;
```

Comment:
The thought was that this might be useful to a customer doing their own step-wise migration.

Reply:
Yes, but once introduced it cannot be removed in basically forever.  Your call.

Reply:
I think that is fine, because "forever" is bounded by the lifetime of `CommitOptions` itself, which we should deprecate along with the other specialty `spanner::*Options` classes when the `Options` migration is complete.